frontend/packages/kubevirt-plugin/integration-tests/tests/dialogs/diskDialog.ts

```diff
@@ -0,0 +1,86 @@
+import { click } from '@console/shared/src/test-utils/utils';
+import { fillInput, selectOptionByText, getSelectedOptionText } from '../utils/utils';
+import * as view from '../../views/dialogs/diskDialog.view';
+import { applyButton, saveButton } from '../../views/kubevirtDetailView.view';
+import { StorageResource, DiskSourceConfig } from '../utils/types';
+import { DISK_SOURCE } from '../utils/consts';
+import { waitForNoLoaders } from '../../views/wizard.view';
+
+export class DiskDialog {
+  sourceMethods = {
+    'Attach Cloned Disk': async (sourceConfig: DiskSourceConfig) =>
+      DiskDialog.selectSourceAttachClonedDisk(sourceConfig),
+    'Attach Disk': async (sourceConfig: DiskSourceConfig) =>
+      DiskDialog.selectSourceAttachDisk(sourceConfig),
+    Container: async (sourceConfig: DiskSourceConfig) =>
```

Comment:
```suggestion
    [DISK_SOURCE.Container]: async (sourceConfig: DiskSourceConfig) =>
```
etc.
looks better

Reply:
Thanks, I was wandering how to do this using the `enum`